src/status_im/contexts/settings/wallet/saved_addresses/sheets/share_address/view.cljs

```diff
@@ -0,0 +1,107 @@
+(ns status-im.contexts.settings.wallet.saved-addresses.sheets.share-address.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [status-im.constants :as constants]
+    [status-im.contexts.wallet.common.utils :as utils]
+    [status-im.contexts.wallet.common.utils.networks :as network-utils]
+    [status-im.contexts.wallet.sheets.network-preferences.view :as network-preferences]
+    [utils.i18n :as i18n]
+    [utils.image-server :as image-server]
+    [utils.re-frame :as rf]))
+
+(def qr-size 500)
+
```

Comment:
Is this qr-size unique to this screen? or is shared for the other qr screens?

Reply:
I believe this size is the same for all share QR screens.

Reply:
yes, i can see the same size everywhere